autometa/common/external/bedtools.py

```diff
@@ -107,15 +118,15 @@ def parse(bed, out=None):
     df = df.assign(total_breadth=lambda x: x.depth * x.bases)
```

Comment:
See Issue #18, point 4. Perhaps `depth_product` would be more descriptive? The problem with `total_breadth` is that I can't tell what it is from the name, and it also is not a total of anything for the contig.

Reply:
I'll switch the column name to `depth_product`